pkg/urbit/daemon/main.c

```diff
@@ -530,23 +594,37 @@ _setup_ssl_curl(void* arg)
 /* _cw_usage(): print utility usage.
 */
 static void
-_cw_usage(c3_c* s)
+_cw_usage(c3_c* bin_c)
 {
-  fprintf(stderr,
-    "\nutilities:\n"
-    "  %s cram <pier>               jam state:\n"
-    "  %s grab <pier>               measure memory usage:\n"
-    "  %s info <pier>               print pier info:\n"
-    "  %s meld <pier>               deduplicate snapshot:\n"
-    "  %s pack <pier>               defragment snapshot:\n"
-    "  %s queu <pier> <at-event>    cue state:\n"
-    "\n  run as a 'serf':\n"
+  c3_c *use_c[] = {
+    "utilities:\n",
+    "  %s cram %.*s              jam state:\n",
+    "  %s dock %.*s              copy binary:\n",
+    "  %s grab %.*s              measure memory usage:\n",
+    "  %s info %.*s              print pier info:\n",
+    "  %s meld %.*s              deduplicate snapshot:\n",
+    "  %s pack %.*s              defragment snapshot:\n",
+    "  %s prep %.*s              prepare for upgrade:\n",
+    "  %s next %.*s              request upgrade:\n",
+    "  %s queu %.*s<at-event>    cue state:\n",
+    "  %s vere ARGS <output dir>    download binary:\n",
```

Comment:
I think it makes some sense to only print the short versions here given the context (cli-level usage docs, instead of a more handhold-y guide/manual).

Should we mention the options/flags too? We have some for `vere` and `next`, right?

Also unsure why there's `:`s at the end of all of these.

Some suggestions for improving clarity below.

```suggestion
    "utilities:\n",
    "  %s cram %.*s              export snapshot:\n",
    "  %s dock %.*s              install binary into pier:\n",
    "  %s grab %.*s              measure memory usage:\n",
    "  %s info %.*s              print pier info:\n",
    "  %s meld %.*s              deduplicate snapshot:\n",
    "  %s pack %.*s              defragment snapshot:\n",
    "  %s prep %.*s              prepare for upgrade:\n",
    "  %s next %.*s              request upgrade:\n",
    "  %s cueu %.*s<at-event>    import snapshot:\n",
    "  %s vere ARGS <output dir>    download binary:\n",
```

("queu" is not a word, so if we're making things up anyway, "cueu" more closely matches the "cue" of "cram"'s "jam". Alternatively, "clue" could be worth considering. Happens to fit nicely with "cramming" in the context of studying, getting a clue to recall things by.)